hw/ip/otbn/data/otbn.hjson

```diff
@@ -33,14 +33,6 @@
       randcount: "256",
       randtype:  "data"
     },
```

Comment:
if you decide to rename the `RndCnstUrndLfsrSeed` parameter, you also have to update the Hjson entry here.